pygmt/src/makecpt.py

```diff
@@ -102,7 +102,7 @@ def makecpt(**kwargs):
         For details on array creation, see
         :gmt-docs:`makecpt.html#generate-1d-array`.
     truncate : list or str
-        ``zlo/zhi``.
+        *zlo/zhi*\ .
         Truncate the incoming CPT so that the lowest and highest z-levels are
         to *zlo* and *zhi*. If one of these equal NaN then we leave that end of
         the CPT alone. The truncation takes place before any resampling. See
```

Comment:
In L114 below this in makecpt.py, ``reverse`` should have **c** as bold and ``truncate`` and ``series`` as code.